hugo/content/blog/2020-07-09-mumble-1-3-2-release.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Mumble 1.3.2 Release Announcement
+author: Robert Adam
+date: 2020-07-09
+categories:
+  - News
+  - Release
+  - Security

```

Comment:
What makes this a security release? I think that’s wrong?

Reply:
Ah yes. This is just from copy&pasting xD